tests/monitoring/vm_monitoring.go

```diff
@@ -255,6 +259,26 @@ var _ = Describe("[Serial][sig-monitoring]VM Monitoring", Serial, decorators.Sig
 			Expect(err).ToNot(HaveOccurred())
 		}
 
+		createVmAndSnapshot := func() *snapshotv1.VirtualMachineSnapshot {
```

Comment:
Is there a reason this is a closure and not a dedicated function?
Can you please separate it into two functions? we try to eliminate the pattern of a function that does more than one thing, in this case the separation is simple, creating a VM and creating a VM snapshot are not that tightly coupled to have them in one func